src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Semantics/Types/Type.cs

```diff
@@ -198,8 +198,18 @@ private static Type CalculateAssociatedSystemTypeForAggregate(AggregateType aggt
         }
 
         // ITypeOrNamespace
-        public bool IsType() { return true; }
-        public bool IsNamespace() { return false; }
+        public bool IsType
+        {
+            get
+            { return true; }
+        }
+
+        public bool IsNamespace
+        {
+            get
+            { return false; }
```

Comment:
Nit: these newlines are unnecessary.  They could also be done with expression-bodied members.